testing-and-tdd/mocking-and-abstraction.md

```diff
@@ -0,0 +1,367 @@
+# Mocking and Abstraction
+
+### Projected Time
+
+About 1 hour and 10 minutes
+
+- Lesson: 30 min
+- Guided Practice: 15 min
+- Independent Practice: 25 min
+- Check for Understanding: 10 min
+
+
+[simplemock-home]: https://www.npmjs.com/package/simple-mock
+[nock-home]: https://github.com/nock/nock
+[nock-intro]: https://scotch.io/tutorials/nodejs-tests-mocking-http-requests
+[backend-ii]: https://glitch.com/edit/#!/zee-techtonica-backend-testing-ii?path=index.js:37:0
```

Comment:
```suggestion
[simplemock-home](https://www.npmjs.com/package/simple-mock)
[nock-home](https://github.com/nock/nock)
[nock-intro](https://scotch.io/tutorials/nodejs-tests-mocking-http-requests)
[backend-ii](https://glitch.com/edit/#!/zee-techtonica-backend-testing-ii?path=index.js:37:0)
```

Reply:
mmm I agree with the rest of the changes except for this one, im not calling a link here, im defining it to use it later on

Reply:
Oh, okay. Then you could leave it like that @pa7ri